.github/ISSUE_TEMPLATE/report-issue.md

```diff
@@ -1,34 +1,42 @@
 ---
-name: Report Issue
-about: Report an issue to help us improve
+name: Report A Helm Chart Issue
+about: Report helm chart issues to help us improve
```

Comment:
Not sure about this sentence, `Report helm chart issues to help us improve`, it feels like its hanging and not complete. @tan-ro any recommendations?